diff --git a/config/Config-build.in b/config/Config-build.in
index e0889fc..a6478fe 100644
--- a/config/Config-build.in
+++ b/config/Config-build.in
@@ -31,7 +31,7 @@
 
 	config SIGNED_PACKAGES
 		bool "Cryptographically signed package lists"
-		default y
+		default n
 
 	comment "General build options"
 
@@ -178,7 +178,7 @@
 	choice
 		prompt "User space Stack-Smashing Protection"
 		depends on USE_MUSL
-		default PKG_CC_STACKPROTECTOR_REGULAR
+		default PKG_CC_STACKPROTECTOR_STRONG
 		help
 		  Enable GCC Stack Smashing Protection (SSP) for userspace applications
 		config PKG_CC_STACKPROTECTOR_NONE
@@ -196,7 +196,7 @@
 
 	choice
 		prompt "Kernel space Stack-Smashing Protection"
-		default KERNEL_CC_STACKPROTECTOR_REGULAR
+		default KERNEL_CC_STACKPROTECTOR_STRONG
 		depends on USE_MUSL || !(x86_64 || i386)
 		help
 		  Enable GCC Stack-Smashing Protection (SSP) for the kernel
