diff --git a/include/target.mk b/include/target.mk
index ef7d8555d8..1361d68769 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -13,11 +13,15 @@ __target_inc=1
 DEVICE_TYPE?=router
 
 # Default packages - the really basic set
-DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci opkg netifd fstools uclient-fetch logd
+DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci opkg netifd fstools uclient-fetch logd \
+luci luci-app-upnp luci-app-samba
 # For nas targets
 DEFAULT_PACKAGES.nas:=block-mount fdisk lsblk mdadm
 # For router targets
-DEFAULT_PACKAGES.router:=dnsmasq iptables ip6tables ppp ppp-mod-pppoe firewall odhcpd odhcp6c
+DEFAULT_PACKAGES.router:=dnsmasq-full iptables ip6tables ppp ppp-mod-pppoe firewall odhcpd odhcp6c \
+iptables-mod-tproxy block-mount kmod-usb-storage kmod-fs-ext4 kmod-fast-classifier kmod-ipt-conntrack-extra \
+luci-app-chinadns luci-app-shadowsocks shadowsocks-libev simple-obfs hdparm luci-app-vlmcsd\
+
 DEFAULT_PACKAGES.bootloader:=
 
 ifneq ($(DUMP),)
